radix-operator/job/handler.go

```diff
@@ -25,31 +26,43 @@ const (
 	MessageResourceSynced = "Radix Job synced successfully"
 )
 
-// Handler Instance variables
-type Handler struct {
+// Handler Common handler interface
+type Handler interface {
+	// Sync Is created on sync of resource
+	Sync(ctx context.Context, namespace, name string, eventRecorder record.EventRecorder) error
+	// CleanupJobHistory Cleanup the pipeline job history
+	CleanupJobHistory(ctx context.Context, radixJob *v1.RadixJob)
+}
```

Comment:
```suggestion
type Handler interface {
	common.Handler
	// CleanupJobHistory Cleanup the pipeline job history
	CleanupJobHistory(ctx context.Context, radixJob *v1.RadixJob)
}
```
"Inherit" the common.Handler instead of redecalring the Sync method

Reply:
Fixed